synapse/app/homeserver.py

```diff
@@ -88,11 +86,10 @@ def build_resource_for_web_client(hs):
                 "the location of the source to serve via the configuration\n"
                 "option `web_client_location`\n\n"
                 "To install the `matrix-angular-sdk` via pip, run:\n\n"
-                "    pip install '%(dep)s'\n"
+                "    pip install matrix-angular-sdk\n"
                 "\n"
                 "You can also disable hosting of the webclient via the\n"
                 "configuration option `web_client`\n"
-                % {"dep": DEPENDENCY_LINKS["matrix-angular-sdk"]}
```

Comment:
The purpose of DEPENDENCY_LINKS seems to be maintaining a list of all external dependencies which needs to be installed via url.
DEPENDENCY_LINKS is not initialized anywhere else afaik.
Also it seems to be static data like REQUIREMENTS & CONDITIONAL_REQUIREMENTS.

For your case you should probably link the "matrix-angular-sdk" from CONDITIONAL_REQUIREMENTS["web_client"]